python/utils/mtx2csv.py

```diff
@@ -0,0 +1,46 @@
+import os
+import time
+from scipy.io import mmread
+import argparse
+
+
+parser = argparse.ArgumentParser(description='Convert the sparsity pattern \
+                                 of a MatrixMarket file into a CSV file. \
+                                 Each directed edge is explicitely stored, \
+                                 edges are unsorted, IDs are 0-based.')
+parser.add_argument('file', type=str,
```

Comment:
If you use `type=argparse.FileType()` then the `file` arg will be a file-like object. And the parser will check that it exists and is readable, plus support using `-` for bash piping.

Reply:
addressed in f3a16b4